staging/src/github.com/kubeedge/beehive/pkg/core/context/context_factory.go

```diff
@@ -40,7 +40,7 @@ func InitContext(contextType string) {
 	})
 }
 
-func GetContext() gocontext.Context{
+func GetContext() gocontext.Context {
```

Comment:
I think this line is gofmt issue? @daixiang0

Reply:
The gofmt linter only lint files from `go list ./...`，unlucky, the `staging` dir is not in this list.
Use `gofmt -l -w ./..` can check `vendor` and `staging` dir.

Reply:
Vendor should not in our check scope..